contracts/router/interfaces/IBridgeModule.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {BridgeToken, SwapQuery} from "../libs/Structs.sol";
+
+interface IBridgeModule {
+    /// @notice Performs a bridging transaction on behalf of the sender, assuming `token` is already in the contract.
+    /// @dev This will be used via delegatecall from SynapseRotuerV2, which will have custody over the initial tokens.
```

Comment:
Small typo here
```suggestion
    /// @dev This will be used via delegatecall from SynapseRouterV2, which will have custody over the initial tokens.
```